### DISCUSSION TITLE
Update Relationship Field

### DISCUSSION THREAD

**Discussion by mikee1209** [Budibase/budibase#4278] [Help] [answered]
Hi, I'm trying to create the form to add an invoice. In this form I have field called Contractor which is Relationship field to contractors table. Is it possible to update DropDownList after adding new contractor in modal?

**Comment by pavleks** [comment]
Hi @mikee1209 I might got your question wrong, so please forgive me if I am answering the wrong question.  I have a "Submit an invoice" form connected to the Contractors table.  In the Contractors table, I have "Contractor" column which is type "Options". I have 4 Options. As soon as I update the Contractor options by adding additional option Painter, the form updates aswell. Please refer to the video. 

https://user-images.githubusercontent.com/76126966/151953701-e367181a-0ce8-444a-be85-53e7472cbaf8.mp4

**Comment by mjashanks** [comment, marked as answer]
Thanks @pavleks

@mikee1209 - I think I get you, you want to reload your contractors dropdown after adding a new record **via a modal**.

The reason that this is difficult is because the modal cannot communicate with the parent screen, i.e. to say "The Row Saved, you can reload now".

Unfortunately, this is a limitation of using modals at the moment. If you navigate to the screen normally (and back) then it will work, but then you lose any data that you have entered.

Not ideal - but definitely something that I think we should address as it does come up.

**Reply by mikee1209** [reply]
Thanks @pavleks and @mjashanks 

The issue I have is exactly like you explained. Thank you for suggestion I will go that way.
